.env.example

```diff
@@ -1,6 +1,6 @@
+APP_DATA_DIR=/Users/greg/Library/Application Support/com.tauri.dev
+PROJECT_NAME=project-x
 OPENAI_API_KEY=
-OPENAI_COMPLETE_MODEL=davinci
```

Comment:
OpenAI recently announced this is being deprecated and we do not use it.

Reply:
@gjreda should I then remove this from the configurations screen and fw to sidecar?

Reply:
@cguedes Yes, they'll be removing it from the API: https://openai.com/blog/gpt-4-api-general-availability